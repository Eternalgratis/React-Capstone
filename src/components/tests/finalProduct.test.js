import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/configureStore';
import FinalProducts from '../finalProduct';

describe('to test the finalProduct components', () => {
  test('matches the finalDetails DOM snapshot', () => {
    const tree = renderer.create(
      <BrowserRouter>
        <Provider store={store}>
          <FinalProducts />
        </Provider>
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
