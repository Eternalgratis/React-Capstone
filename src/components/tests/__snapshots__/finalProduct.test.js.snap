// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`to test the finalProduct components matches the finalDetails DOM snapshot 1`] = `
<a
  href="/product/undefined"
  onClick={[Function]}
>
  <div
    className="main-container"
  >
    <div
      className="prod-cont"
    >
      <img
        height="200px"
        width="200px"
      />
      <p />
      <h1 />
    </div>
  </div>
</a>
`;
